pkg/healthsrv/healthz_handler_test.go

```diff
@@ -45,8 +46,37 @@ func TestHealthZBucketListErr(t *testing.T) {
 
 func TestReadinessNamespaceListErr(t *testing.T) {
 	nsLister := errNamespaceLister{err: errTest}
+	client := successGetClient{}
+	os.Setenv("DEIS_CONTROLLER_SERVICE_HOST", "127.0.0.1")
+	os.Setenv("DEIS_CONTROLLER_SERVICE_PORT", "8000")
```

Comment:
I'm always a bit wary of setting env vars inside of tests because it's messing with global state. But, I see that `ControllerURLStr` requires them. It's not blocking for this PR to change this, but can you create an issue to refactor this and the implementation of `ControllerURLStr` also?